### ISSUE TITLE
How to control the model's bpp?

### ISSUE THREAD

**Issue by hot-dog** [InterDigitalInc/CompressAI#185]
First of all, thanks for your excellent work! I am new to image compression field and i have got a question.
I want to train a model with bpp approximatly at 0.8, what should i do? Can i do this by setting the training hyper-parameters such as quality, lambda? Is there a formula between model's bpp and these training hyper-parameters?

**Comment by YodaEmbedding** [comment]
[These lambda values](https://interdigitalinc.github.io/CompressAI/zoo.html#training) should give you an approximate guideline.

Typically, most models trained from scratch quickly converge to their final bpp values (+/- 5%) within the first 1-10 epochs.

If you want to target a specific bpp with more precision than this, take a pretrained model, and then reduce it's lambda value by a bit, and train for ~1 epoch. The resulting bpp value should be within +/-2% of the final bpp. Repeat as desired.

**Comment by fracape** [comment]
Since you don't mention if you have a specific image / test set for which you need an encode at 0.8bpp, also note that the bitrate is very dependent on the content. The above is true if you keep Kodak as your reference test set for instance. You may want to check with your dataset if different.
Even with Kodak, the figure below shows you how different the bpp and qualities can be using the same lambda among the images of Kodak (light blue). The orange dot is the average on Kodak for that particular lambda. 
<img width="421" alt="image" src="https://user-images.githubusercontent.com/67165941/208851458-d4675ee0-6de4-43e5-ac39-05a9cc5f85a6.png">

**Comment by hot-dog** [comment]
Thanks for your explanation, i get it :). Issue closed!
